docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -697,8 +791,531 @@ To read and write a value of the Bluetooth descriptor:
+   ```
+   > [!NOTE]
+   > The service will be unregistered from the server when:
+   > - service's `unregister()` method is called,
+   > - server's `unregisterAllServices()` method is called,
+   > - application is reloaded,
+   > - application is closed.
+
+### Unregister services
+GATT services can be unregistered either one at a time or all at once.
+
+To unregister a single service from the local GATT server:
+
+1. Choose the service to be unregistered:
+   ```
+   var server = tizen.bluetooth.getGATTServer();
+   var serviceToBeUnregistered = server.services[3]; 
+   ```
+
+2. Call `unregister()` method:
+   ```
+   function successCallback() {
+     console.log('Service unregistered successfully');
+   }
+
+   function errorCallback(error) {
+     console.log('Unregistering GATT service failed: ' + error.message);
+   }
+
+   serviceToBeUnregistered.unregister(successCallback, errorCallback);
+   ```
+   > [!NOTE]
+   > After `unregister()` is called for the last registered service, the server is stopped.
+   > If new services are then registered, server's `start()` method has to be
+   > called to make them visible to clients.
+
+To unregister all services from the local GATT server at once, call server's
+`unregisterAllServices()` method:
+
+   ```
+   function successCallback() {
+     console.log('All services unregistered successfully');
+   }
+
+   function errorCallback(error) {
+     console.log('Unregistering all GATT services failed: ' + error.message);
+   }
+
+   var server = tizen.bluetooth.getGATTServer();
+   server.unregisterAllServices(successCallback, errorCallback);
+   ```
+   > [!NOTE]
+   > After the calling `unregisterAllServices()`, the server is stopped.
+   > If new services are then registered, server's `start()` method has to be
+   > called to make them visible to clients.
+
+### Send notifications about characteristic's value changes to the clients
+GATT clients connected to the server running on the local device can register for updates of its characteristics' values.
+The server can send two types of such updates - _notifications_ and _indications_.
+They differ in that, clients receiving indications have to acknowledge them - i.e. send a message back to the server, telling that they received the new value. Clients do not acknowledge notifications.
+
+Notifications and indications are not enabled in characteristics by default.
+To enable notifications or indications in a characteristic:
+
+1. Set its `isNotify` or `isIndication` property:
+
+   ```
+   var notificationEnabledCharacteristicInitData = {
+     uuid: '1234',
+     isNotify: true // or set isIndication to enable indications
+     /* Define more attributes, if needed */
+   };
+   ```
+2. The characteristic requires a special kind of descriptor, called _Client Characteristic Configuration Descriptor_ (CCCD).
+For more details about CCCD, refer to the _Bluetooth Core Specification_, available at [bluetooth.com](https://bluetooth.com).
```

Comment:
@surbhig12 Would you please check again these sentences?

```suggestion
2. The characteristic requires a special kind of descriptor, Client Characteristic Configuration Descriptor (CCCD).
For more details about CCCD, see [Bluetooth Core Specification](https://www.bluetooth.com/specifications/bluetooth-core-specification/).
```
@pwasowski2 I hope you to check the updated link also.

Reply:
The link is OK. I have applied your suggestions.